### ISSUE TITLE
node js instation error kindly needfully ASAP

### ISSUE THREAD

**Issue by rpavanfour** [nodejs/help#3789]
### Details

node js instation error kindly needfully ASAP

error msgs shows

```console
Is this OK? (yes) yes
npm ERR! code EPERM
npm ERR! syscall open
npm ERR! path C:\Users\cypressautomation\package.json
npm ERR! errno -4048
npm ERR! Error: EPERM: operation not permitted, open 'C:\Users\cypressautomation\package.json'
npm ERR!  [Error: EPERM: operation not permitted, open 'C:\Users\cypressautomation\package.json'] {
npm ERR!   errno: -4048,
npm ERR!   code: 'EPERM',
npm ERR!   syscall: 'open',
npm ERR!   path: 'C:\\Users\\cypressautomation\\package.json'
npm ERR! }
npm ERR!
npm ERR! The operation was rejected by your operating system.
npm ERR! It's possible that the file was already in use (by a text editor or antivirus),
npm ERR! or that you lack permissions to access it.
npm ERR!
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\RASAMALLA PAVANKUMAR\AppData\Local\npm-cache\_logs\2022-03-17T17_23_09_607Z-debug-0.log
```

### Node.js version

Not applicable.
Download for Windows (x64)
16.14.1 LTS
Recommended For Most Users

and

Latest LTS Version: 16.14.1 (includes npm 8.5.0)

LTS
Recommended For Most Users
Current
Latest Features
Windows Installer
node-v16.14.1-x64.msi

[](url)

### Operating system

Windows 11

### Scope

```console
Is this OK? (yes) yes
npm ERR! code EPERM
npm ERR! syscall open
npm ERR! path C:\Users\cypressautomation\package.json
npm ERR! errno -4048
npm ERR! Error: EPERM: operation not permitted, open 'C:\Users\cypressautomation\package.json'
npm ERR!  [Error: EPERM: operation not permitted, open 'C:\Users\cypressautomation\package.json'] {
npm ERR!   errno: -4048,
npm ERR!   code: 'EPERM',
npm ERR!   syscall: 'open',
npm ERR!   path: 'C:\\Users\\cypressautomation\\package.json'
npm ERR! }
npm ERR!
npm ERR! The operation was rejected by your operating system.
npm ERR! It's possible that the file was already in use (by a text editor or antivirus),
npm ERR! or that you lack permissions to access it.
npm ERR!
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\RASAMALLA PAVANKUMAR\AppData\Local\npm-cache\_logs\2022-03-17T17_23_09_607Z-debug-0.log
```

### Module and version

Not applicable.

**Comment by VoltrexMaster** [comment]
The error is self-explanatory here.

```console
npm ERR! The operation was rejected by your operating system.
npm ERR! It's possible that the file was already in use (by a text editor or antivirus),
npm ERR! or that you lack permissions to access it.
npm ERR!
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.
```

You're missing root/Administrator permissions to install NPM packages, as you seem to be using Windows, this can be resolved by opening a Command Prompt, Powershell, or Windows Terminal window as Administrator.

Here is a guide on how you can do that shown [here](https://allthings.how/how-to-open-windows-terminal-as-admin-on-windows-11/).

**Comment by rpavanfour** [comment]
[image: image.png]

kindly show SCREEN needful ASAP.

**Comment by takipsizad** [comment]
it explains itself? permissions is the error

**Comment by future-media** [comment]
This occurs when installing node 14.21.3 on a clean Windows system and then attempting to update npm to the latest (9.7.1) version. It can be resolved by first updating npm to version 7.24.2 and then to latest.
